Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift

```diff
@@ -19,11 +19,13 @@ struct SwiftUINativeCheckoutDemo: View {
     var body: some View {
         switch viewModel.state {
         case .initial:
-            getAccessTokenView()
+            getClientIDView()
         case .loading(let content):
             loadingView(content)
         case let .mainContent(title, content, isFlowComplete):
             checkoutView(title, content, isFlowComplete)
+        case .error(message: let message):
```

Comment:
nit: can we do this here to match the other cases?
```suggestion
        case .error(let message):
```